### DISCUSSION TITLE
Welcome to vaccine-feed-ingest Discussions!

### DISCUSSION THREAD

**Discussion by obra** [CAVaccineInventory/vaccine-feed-ingest#77] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by szilbert** [comment]
Hello,

I saw patio11's [post on HN](https://news.ycombinator.com/item?id=26920732) and was inspired to contribute. What a great project!

While I was setting up my development environment so that I could start working on Issues, I ran into a few speedbumps and also saw a few things that could be tweaked - things like typos in commands in the wiki or documentation pages, additional unlisted prerequisites that were needed for manual development environment setup on my system, and warnings about yml files not being marked as executable.

Would it be helpful to have new Issues and Pull requests for those kinds of changes, or would that be a distraction from the work of the project?

**Reply by obra** [reply]
Absolutely!
